mllib/src/main/scala/org/apache/spark/ml/tree/impl/GradientBoostedTrees.scala

```diff
@@ -329,8 +330,16 @@ private[spark] object GradientBoostedTrees extends Logging {
       //       However, the behavior should be reasonable, though not optimal.
       baseLearnerWeights(m) = learningRate
 
+      if (pre_predError.getStorageLevel != StorageLevel.NONE ){
```

Comment:
Spacing is wrong at the end of the line -- see the style guide.

is predError definitely already consumed and not used at this point?